tests/unit/services/fractie-api-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'frontend-lmb/tests/helpers';
+
+module('Unit | Service | fractie-api', function (hooks) {
+  setupTest(hooks);
+
+  // TODO: Replace this with your real tests.
+  test('it exists', function (assert) {
+    let service = this.owner.lookup('service:fractie-api');
+    assert.ok(service);
+  });
```

Comment:
Do we keep these or can I just remove it?

Reply:
You can leave it, all are tests right now are a bit stupid. But personally I never add these test files, since they don't really do anything.